addons/mail/static/src/views/fields/properties_field/property_value.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { PropertyValue } from "@web/views/fields/properties/property_value";
+import { TagsList } from "@web/core/tags_list/tags_list";
+import { useOpenChat } from "@mail/web/open_chat_hook";
+
+// Many2one
+patch(PropertyValue.prototype, "mail.PropertyValue", {
+    setup() {
+        this._super();
+
+        if (this.env.services["mail.thread"]) {
+            // work only for the res.users model
+            this.openChat = useOpenChat("res.users");
+        }
+    },
+
+    _onAvatarClicked() {
+        if (this.openChat && this.showAvatar && this.props.comodel === "res.users") {
+            this.openChat(this.props.value[0]);
+        }
+    },
+});
+
+// Many2many
+export class Many2manyPropertiesTagsList extends TagsList {
+    static template = "mail.Many2manyPropertiesTagsList";
+
+    setup() {
+        super.setup();
+        if (this.env.services["mail.thread"]) {
+            this.openChat = useOpenChat("res.users");
+        }
+    }
+
+    _onAvatarClicked(tagIndex) {
+        const tag = this.props.tags[tagIndex];
+        if (this.openChat && tag.comodel === "res.users") {
```

Comment:
There is no "not clickable" behavior for tagslist (aka it is considered as always 'showAvatar') ?